azext_iot/tests/iothub/core/test_iot_messaging_int.py

```diff
@@ -836,6 +836,16 @@ def test_hub_monitor_events(self):
             ],
         )
 
+        # Monitor events for all devices, limiting monitor message count to 10
+        num_messages = 10
+        monitor_stop_msgs = ["Successfully parsed {} message(s).".format(num_messages), "Stopping event monitor..."]
+        self.command_execute_assert(
+            "iot hub monitor-events -n {} -g {} --cg {} --et {} --message-count {} -t 8 -y -p sys anno app".format(
```

Comment:
I'm thinking we should omit `-t` (and use the default infinity placeholder 0) since the monitor should terminate via --message-count and it would be more explicit how it was terminated.